app/assets/v2/js/pages/bulk_payout.js

```diff
@@ -185,14 +185,13 @@ $(document).ready(function($) {
 });
 
 var get_total_cost = function() {
-  var num_rows = $('#payout_table tbody').find('tr').length;
+  var rows = $('#payout_table tbody tr');
   var total = 0;
-  var i = 1;
 
-  for (i = 1; i < num_rows; i += 1) {
-    var $row = $('#payout_table tobdy').find('tr:nth-child(' + i + ')');
-    var amount = parseFloat($row.find('.amount').text());
-    var username = $row.find('.username-search').text();
+  for (i = 0; i < rows.length; i += 1) {
```

Comment:
Maybe we want to use let / ES5/6 style ? cc @thelostone-mc @SaptakS 
Like:
```
for (let i = 1; i < num_rows; i++) {
  let row = rows[i]; // XXX: Do we need $(rows[i])?
```

Reply:
I think Gitcoin is moving towards ES6 styling if I remember correctly, so I'd be glad to switch these over.

Reply:
It's okay for now. We will anyways do a complete migration from old-school to ES6 soon.